src/TConsole.h

```diff
@@ -298,8 +320,57 @@ public slots:
 
 
     ConsoleType mType;
+
+    // Was public in Host class but made private there and cloned to here
+    // (for main TConsole) to prevent it being changed without going through the
+    // process to load in the the changed dictionary:
```

Comment:
I don't understand this comment - why do the prevention?

Reply:
Changing that value needs to do more than just changing that value - forcing the preference dialogue to change it via `Host::setSpellDic(const QString&)` ensures that the appropriate dictionary is loaded via a call to `TConsole::setSystemSpellDictionary(const QString&)` for the main `TConsole`.

Reply:
Ok - makes sense - could you revise it to be simpler to understand, ie "changing this value must also trigger that action"?